tests/python/gaia-ui-tests/gaiatest/apps/bzlite/app.py

```diff
@@ -20,16 +30,59 @@ def login(self, username, password):
         password_element.tap()
         password_element.send_keys(password)
         button_element = self.marionette.find_element(*self._button_login)
-        button_element.tap()   
+        button_element.tap()
+
+    def logout(self):
+        profile_element = self.marionette.find_element(*self._profile_icon)
+        profile_element.tap()
+        logout_element = self.marionette.find_element(*self._button_logout)
+        logout_element.tap()
+
+    def followBugs(self):
```

Comment:
This is probably a leftover from another PR, but this method is not used. Let's remove it.